indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -96,8 +96,23 @@
                                 </td>
                                 {% for item in static_columns %}
                                     {% if item.id == 'state' %}
-                                        <td class="i-table" data-searchable="{{ abstract.state.title | lower }}">
-                                            {{- abstract.state.title -}}
+                                        <td class="i-table" data-searchable="{{ abstract.public_state.title | lower }}">
+                                            <div class="i-tag outline semantic-text {{ abstract.public_state.css_class }} state-badge">
```

Comment:
Maybe this could be moved inside a macro? Not that we're going to reuse it, but at least it doesn't look like a gigantic list of ifs and elses.